resources/views/account/partials/register-account.blade.php

```diff
@@ -37,6 +37,29 @@
                 v-model="variables.password"
                 required
             />
+            <toggler>
+                <div slot-scope="{ toggle, isOpen }" class="contents">
+                    <x-rapidez-ct::input.checkbox
+                        id="isb2b"
+                        name="isb2b"
+                        v-model="isOpen"
+                        v-on:click="toggle"
+                    >
+                        @lang('This is a business account')
+                    </x-rapidez-ct::input.checkbox>
+                    @if(config('customer/create_account/vat_frontend_visibility', 0))
```

Comment:
It may be nice to place this around the entire toggler considering it is the only field used in the input and i do not see another value in the graphql request for creating a customer that could be put in here